gamemode/core/libs/thirdparty/sh_tween.lua

```diff
@@ -263,33 +263,33 @@ end
 
 local function checkSubjectAndTargetRecursively(subject, target, path)
   path = path or {}
-  local targetType, newPath
+  local newPath
   for k,targetValue in pairs(target) do
-    targetType, newPath = type(targetValue), copyTables({}, path)
+    newPath = type(targetValue), copyTables({}, path)
```

Comment:
Forgot to remove the `targetType` variable assignment

Reply:
Yep i commented on it few seconds ago